bindings/node.js/lib/kzg.js

```diff
@@ -31,19 +53,49 @@ function transformTrustedSetupJson(filePath) {
   return outputPath;
 }
 
-const originalLoadTrustedSetup = bindings.loadTrustedSetup;
-// docstring in ./kzg.d.ts with exported definition
-bindings.loadTrustedSetup = function loadTrustedSetup(filePath) {
-  if (!(filePath && typeof filePath === "string")) {
-    throw new TypeError("must initialize kzg with the filePath to a txt/json trusted setup");
-  }
-  if (!fs.existsSync(filePath)) {
-    throw new Error(`no trusted setup found: ${filePath}`);
+/**
+ * Gets location for trusted setup file. Uses user provided location first. If
+ * one is not provided then defaults to the official Ethereum mainnet setup from
+ * the kzg ceremony.
```

Comment:
Nit: in the new code/comments, can we replace "kzg" with "KZG"? Except for "c-kzg" of course.
```suggestion
 * the KZG ceremony.
```

Reply:
Updated it here and in all other locations "kzg" was in the docs and docstrings.